docs/headers.md

```diff
@@ -69,6 +69,7 @@ func (h *myServer) MyRPC(ctx context.Context, req *pb.Req) (*pb.Resp, error) {
   return &pb.Resp{}, nil
 }
 ```
+You can also use `twirp.AddHTTPResponseHeader` to add values to the key that is already exists.
```

Comment:
Minor typo here, I suggest:
`twirp.AddHTTPResponseHeader` can be used to append values to a header.

Reply:
Done👍
https://github.com/twitchtv/twirp/pull/227/commits/dcfe73704d81e9f2849dea3da9d007230b2d1e79